app/assets/javascripts/assignment_swap/validateTwoAssignmentsSwapping.js

```diff
@@ -68,4 +68,4 @@ function add_events()
   }
 }
 
-window.onload = function(){ add_events(); }
+window.onload = check_for_two_assignments_selected
```

Comment:
Better name! Function names should also be camel case. I was including them in variable names (they're all just objects).